release-to-candidate/action.yaml

```diff
@@ -169,8 +169,21 @@ runs:
       env:
         SNAPCRAFT_STORE_CREDENTIALS: ${{ inputs.store-token }}
         SNAP_FILE: ${{ steps.build.outputs.snap }}
+        components: ${{ steps.snapcraft-yaml.outputs.components }}
+        name: ${{ steps.snapcraft-yaml.outputs.snap-name }}
       run: |
-        snapcraft_out="$(snapcraft push "$SNAP_FILE" --release="${{ inputs.channel }}")"
+        component_flags=()
+        for comp in ${components//,/ }
+        do
+            comp_name=$(echo "$comp" | cut -f1 -d"|")
```

Comment:
Can you quote the instances of `$()` throughout this block?

Reply:
Done